package.json

```diff
@@ -12,15 +12,16 @@
     "build": "gulp build",
     "contributors:add": "all-contributors add",
     "contributors:generate": "all-contributors generate",
+    "format": "prettier --write **/*.js",
```

Comment:
This currently is only checking one level in. Can we write a path glob that gets all of our source files?

We'll also want to ignore `dist/`

Reply:
@jwu910  The path gets all js sources in the project not only the first level in.
                  The dist and node_modules directories are added to .prettierignore

Reply:
Maybe I missed something. I made some changes to files inside src/utils, and they were not corrected with `npm run format`.

Were you able to get it working?

Reply:
Hmm thats weird,i just checked it and prettier is applied on all directories

Reply:
I just found that the glob needs to be wrapped in quotes. What OS are you on?

Reply:
Done already